src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -726,11 +703,13 @@ private void removeTerritoriesWhereTransportsAreExposed(final Map<Territory, Pro
               defenders.addAll(moveMap.get(unloadTerritory).getMaxUnits());
             }
             final ProBattleResult result =
-                battleUtils.calculateBattleResults(player, unloadTerritory, enemyAttackOptions.getMax(unloadTerritory)
-                    .getMaxUnits(), new ArrayList<Unit>(defenders), new HashSet<Unit>(), false);
+                ProBattleUtils.calculateBattleResults(player, unloadTerritory,
```

Comment:
Minor things add up. The extra arguments on `calculateBattleResults` make the it hard to use the method, and makes it harder to understand the usages of the method. IE: if you're writing code, figuring out how to use it is difficult, what does the boolean do? What is this emtpy hashSet? And the same questions are asked by readers of the code too. Boolean args are low hanging fruit in that effort